docs/getting_started.md

```diff
@@ -28,6 +28,10 @@ kubectl create namespace argocd
 kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/core-install.yaml
 ```
 
+!!! note 
+    Default namespace for Kubectl config must be set to `argocd`:
+    `kubectl config set-context --current --namespace=argocd`
+
```

Comment:
Mention that this is only needed for the following commands since the previous commands have `-n argocd` already.

Also change: Kubectl -> `kubectl`